manifests/state-ofed-driver/0050_ofed-driver-ds.yaml

```diff
@@ -135,6 +143,47 @@ spec:
             initialDelaySeconds: {{ .CrSpec.ReadinessProbe.InitialDelaySeconds }}
             failureThreshold: 1
             periodSeconds: {{ .CrSpec.ReadinessProbe.PeriodSeconds }}
+        {{- if .RuntimeSpec.UseDtk }}
+        - image: {{ .RuntimeSpec.DtkImageName }}
+          imagePullPolicy: IfNotPresent
+          name: openshift-driver-toolkit-ctr
+          command: [bash, -xc]
+          args: ["until [ -f /mnt/shared-nvidia-driver-toolkit/dir_prepared ]; do echo  Waiting for nvidia-driver-ctr container to prepare the shared directory ...; sleep 10; done; exec /mnt/shared-nvidia-driver-toolkit/ocp_dtk_entrypoint dtk-build-driver"]
+          securityContext:
+            privileged: true
+            seLinuxOptions:
+              level: "s0"
+          env:
```

Comment:
nit: maybe wrap the `env:` into the if condition to avoid generating `env: {}`

Reply:
Done

Reply:
style suggestion for (imo) easier reading:

```yaml
args:
  - |
    until [ -f /mnt/shared-doca-driver-toolkit/dtk_start_compile ]; do
      echo Waiting for mofed-container container to prepare the shared directory
      sleep 3
    done
    exec /mnt/shared-doca-driver-toolkit/dtk_nic_driver_build.sh
```